app/views/checkout/payment.scala.html

```diff
@@ -60,7 +60,22 @@
                 </fieldset>
                 <div class="u-display-from-tablet" data-set="checkout-notices">
                     <div class="js-append">
-                        @fragments.checkout.notices()
+                        <div class="js-payment-type-direct-debit">
+                            @fragments.checkout.notices()
+                        </div>
+                        <div class="js-payment-type-card" hidden="true">
+                            <div class="notice">
+                                <h3 class="notice__header">First payment</h3>
+                                <div class="notice__content">You first payment will be taken in 14 days under the name Guardian News & Media.</div>
+                            </div>
+                            <div class="notice">
+                                <h3 class="notice__header">Ongoing payments</h3>
+                                <div class="notice__content">Your card will be charged
+                                    @products.find(_.frequency == BillingFrequency.Month).map {product => <span class="js-option-mirror-value">@product.price.pretty every month</span>}.
```

Comment:
Yes, this isn't great. I was copying this, which is used above to create the billing frequency options:

```scala
<fieldset class="basket-billing-options is-hidden js-payment-frequency js-option-mirror-group" id="change-payment-frequency">
    @products.find(_.frequency == BillingFrequency.Month).map { product =>
        @priceOption(product, "every month", product.frequency.numberOfMonths, isChecked=true)
    }
    @products.find(_.frequency == BillingFrequency.Quarter).map { product =>
        @priceOption(product, "every 3 months", product.frequency.numberOfMonths)
    }
    @products.find(_.frequency == BillingFrequency.Annual).map { product =>
        @priceOption(product, "every 12 months", product.frequency.numberOfMonths)
    }
</fieldset>
```
This would suffer from the same issue - if there is no product with a frequency of ```BillingFreqency.month```, the default checked option will not be rendered in the template.

I guess checking outside of the template that we have all expected billing frequencies would be a better option? And throw an error if not?